src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius, String color) {
+        this.radius = radius;
+        super.setColor(color);
```

Comment:
let's use a constructor here, instead of the setter to pass color to the parent class